packages/frontend/core/src/components/affine/auth/send-email.tsx

```diff
@@ -122,7 +122,11 @@ const useSendEmail = (emailType: AuthPanelProps<'sendEmail'>['emailType']) => {
         return trigger({
           email,
           callbackUrl: `/auth/${callbackUrl}?isClient=${
```

Comment:
AppSchema is not needed here?

Reply:
Seems `client` is not used in `/auth/:authType`